props/saber_fett263_buttons.h

```diff
@@ -4400,6 +4408,7 @@ SaberFett263Buttons() : PropBase() {}
   }
 #endif
 
+#ifndef FETT263_DISABLE_QUOTE_PLAYER
```

Comment:
You can simplify this a lot by doing:

void CHeckQuote() {
#ifndef FETT263_DISABLE_QUOTE_PLAYER
    // quote player code
#else
    SaberBase::DoEffect(EFFECT_FORCE, 0);
#endif
}

Then leave all the places you call CheckQuote alone.

Reply:
thanks, updated